webui/react/src/components/XAxisFilter.tsx

```diff
@@ -0,0 +1,33 @@
+import { Select } from 'antd';
```

Comment:
Can you move this into the proper location? Should probably be `/components/kit/LineChart/XAxisFilter.tsx`

Reply:
✅